test/support/conn_case.ex

```diff
@@ -20,7 +20,8 @@ defmodule SkateWeb.ConnCase do
   using do
     quote do
       # Import conveniences for testing with connections
-      use Phoenix.ConnTest
+      import Plug.Conn
+      import Phoenix.ConnTest
```

Comment:
suggestion: you could also inport `Plug.Test` here if you didn't want to make the `Plug.Test.init_session` change.

Reply:
Even better - it looks like `Phoenix.ConnTest` now has its own `init_test_session/2` that's just [delegated to `Plug.Test`](https://github.com/phoenixframework/phoenix/blob/v1.6.6/lib/phoenix/test/conn_test.ex#L253), so I can avoid the direct use of `Plug.Test` entirely.